LICENSE.md

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2017 Barclays Africa Group Limited
+   Copyright 2017 ABSA Group Limited
```

Comment:
It should be `Copyright [yyyy] [name of copyright owner]`

Reply:
Oh, verbatim, not replacing the values?! Sorry, I did not get that.